QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -154,10 +154,10 @@ protected void autonomousMain() throws AutonomousModeOverException, InterruptedE
                     waitForTime(200);
                     Logger.info("Jolting down...");
                     // first, go down momentarily, so if we're already at the top past the limit switch, we won't break anything
-                    height.set(-1.0f); // make sure to go down even though not calibrated.
+                    height.set(-2.0f); // make sure to go down even though not calibrated.
                     mode.set(MODE_HEIGHT);
                     waitForTime(downwardTime);
-                    height.set(2.0f); // make sure to go up even though not calibrated.
+                    height.set(3.0f); // make sure to go up even though not calibrated.
```

Comment:
There is still a chance that this wouldn't reach the top (if distance is really low). Perhaps set the speed and wait for the top limit switch?